package.json

```diff
@@ -796,10 +796,10 @@
         "webpack-cli": "^4.6.0"
     },
     "dependencies": {
```

Comment:
I think we need to remove `vscode-azurekudu` from this list? (`npm remove vscode-azurekudu`)

Reply:
Thanks for your advices, I've made `vscode-azurekudu` as dev dependency